kousa/config/config.exs

```diff
@@ -3,6 +3,10 @@ use Mix.Config
 config :kousa, ecto_repos: [Beef.Repo]
 config :kousa, max_room_size: 1000
 config :extwitter, :json_library, Poison
+config :amqp, connections: [
+  conn: [url: Application.get_env(:kousa, :rabbit_url, "amqp://guest:guest@localhost")],
```

Comment:
yeah. this looks kinda sketch. I just tried to a/b test these changes and I'm not seeing any improvements around reconnecting to mq.  I'll remove the Elixir stuff and let more seasoned hands work with this.  Sorry.

On both occasions, I'm seeing this error message when I 
1) shutdown RabbitMQ while the server's running
2) start it back up
3) and then refresh the page to destroy a room or create a room:
```
23:35:27.988 [error] GenServer :":voice_rabbit" terminating
** (stop) exited in: :gen_server.call(#PID<0.1275.0>, {:call, {:"basic.publish", 0, "", "shawarma_queue", false, false}, {:amqp_msg, {:P_basic, :undefined, :undefined, :undefined, 1, :undefined, :undefined, :undefined, :undefined, :undefined, :undefined, :undefined, :undefined, :undefined, :undefined}, "{\"uid\":\"ec0ef33c-3785-4858-a9c3-87a5f383cf2c\",\"op\":\"close-peer\",\"d\":{\"roomId\":\"f9e1faf3-ae2e-4970-b1fa-7f80ee5f6e0f\",\"peerId\":\"ec0ef33c-3785-4858-a9c3-87a5f383cf2c\"}}"}, #PID<0.1261.0>}, 70000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
    (stdlib 3.14) gen_server.erl:246: :gen_server.call/3
    (amqp 1.6.0) lib/amqp/basic.ex:86: AMQP.Basic.publish/5
    (kousa 0.1.0) lib/gen/voice_rabbit.ex:48: Kousa.Gen.VoiceRabbit.handle_cast/2
    (stdlib 3.14) gen_server.erl:689: :gen_server.try_dispatch/4
    (stdlib 3.14) gen_server.erl:765: :gen_server.handle_msg/6
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Last message: {:"$gen_cast", {:send, "{\"uid\":\"ec0ef33c-3785-4858-a9c3-87a5f383cf2c\",\"op\":\"close-peer\",\"d\":{\"roomId\":\"f9e1faf3-ae2e-4970-b1fa-7f80ee5f6e0f\",\"peerId\":\"ec0ef33c-3785-4858-a9c3-87a5f383cf2c\"}}"}}
23:35:27.990 [error] gen_server ':voice_rabbit' terminated with reason: no such process or port in call to gen_server:call(<0.1275.0>, {call,{'basic.publish',0,<<>>,<<"shawarma_queue">>,false,false},{amqp_msg,{'P_basic',undefined,undefined,...},...},...}, 70000) in gen_server:call/3 line 246
23:35:27.991 [error] CRASH REPORT Process ':voice_rabbit' with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1275.0>, {call,{'basic.publish',0,<<>>,<<"shawarma_queue">>,false,false},{amqp_msg,{'P_basic',undefined,undefined,...},...},...}, 70000) in gen_server:call/3 line 246
```
This leads me to believe that it's related to `Kousa.RegUtils.lookup_and_cast(__MODULE__, id, {:send, Poison.encode!(msg)})` in `Kousa.Gen.VoiceRabbit:send` no longer being able to find the VoiceRabbit genServer.  I'm not sure why the genServer itself would entirely go away since I never saw a message around the genServer terminating prior to step 3) above.  The only time I consistently **do** see the genServer terminating message is after I go through step 3).